molgenis-data-rest/src/test/java/org/molgenis/data/rest/RestControllerIT.java

```diff
@@ -167,7 +167,7 @@ public void deleteWithoutWritePermissionFails()
 			   .then()
 			   .statusCode(UNAUTHORIZED)
 			   .body("errors.message[0]",
-					   equalTo("No [WRITE] permission on entity type [Group authority] with id [sys_sec_GroupAuthority]"));
+					   equalTo("No [READ] permission on entity type [Group authority] with id [sys_sec_GroupAuthority]"));
```

Comment:
Shouldn't this just stay WRITE? Since the function is named: deleteWithoutWritePermissionFails

Reply:
In order to be able to CREATE/UPDATE/DELETE 'Group authority' rows, you require READ access on the metadata.